### ISSUE TITLE
flake8 dependency issue with pyflakes

### ISSUE THREAD

**Issue by cschindlbeck** [ChrisRBe/PP-P2P-Parser#540]
Dependency issue when installing via Python 3.8.13 via 

`
pip install -r requirements.txt 
`

results in 

```
The conflict is caused by:
    The user requested pyflakes==2.4.0
    flake8 3.9.1 depends on pyflakes<2.4.0 and >=2.3.0
```

I have manually fixed the dependecies in https://github.com/cschindlbeck/PP-P2P-Parser/tree/fix-python-3.8.13-dependencies 

This could also be related to https://github.com/ChrisRBe/PP-P2P-Parser/issues/490

**Comment by ChrisRBe** [comment]
Hi @cschindlbeck,
yes. This looks like a duplicate to #490 to me as well. I was playing with the idea to completely drop the support for the requirements.txt in general. 
But since I haven't come to a decision :(  I think there is a problem with the dependabot management... seems that not all packages and dependencies are taken into account when updating the requirements.txt.

**Comment by ChrisRBe** [comment]
Hi @cschindlbeck I created #543 to address this issue. I used pipenv to update to the same versions as per the included Pipfile and afterwards ran pip freeze to update the requirements.txt file.

**Comment by cschindlbeck** [comment]
is resolved via #543
